node/src/components/consensus/protocols/highway.rs

```diff
@@ -479,6 +480,7 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
 enum HighwayMessage<C: Context> {
     NewVertex(Vertex<C>),
     RequestDependency(Dependency<C>),
+    RequestLatestState(Panorama<C>),
```

Comment:
I know you just follow the naming convention but I think that `LatestStateRequest` suits better – it's not an event that is asking this component to request the latest state but it _is_ a request for the latest state.

Reply:
Ping @fizyk20